themes/OpenFisca-2022/layouts/_default/packages.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="{{ $.Site.Language }}">
+
+    {{ partial "head.html" . }}
+
+    <body>
+        {{ partial "sprite.svg" . }}
+
+        {{ partial "header.html" . }}
+
+        {{ partial "hero.html" (dict "title" .Params.hero.title "subtitle" .Params.hero.subtitle "center" true ) . }}
+
+         <div class="wrapper">
+            <div class="grid grid-hasPaddingBottom">
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th>{{ i18n "packages.jurisdiction" }}</th>
+                            <th>{{ i18n "packages.title" }}</th>
+                            <th>{{ i18n "packages.website" }}</th>
+                            <th>{{ i18n "packages.source" }}</th>
+                            <th>{{ i18n "packages.legislation_explorer" }}</th>
+                            <th>{{ i18n "packages.openapi_spec" }}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {{ range $.Site.Data.packages }}
+                            <tr>
+                                <td data-column="{{ i18n "packages.jurisdiction" }}">
+                                    {{ if eq .jurisdiction "ZZ" }}
+                                        <span style="color:var(--colorGray500)">…</span>
```

Comment:
```suggestion
                                        <span style="color:var(--colorGray500)">🏳️</span>
```
…or the `<§>` small logo? 😃